README.md

```diff
@@ -294,6 +294,12 @@ A complete list of commands and flags can be found in the SDK docs at https://sp
   speechmatics batch transcribe --summarize --output-format json-v2 example_audio.wav
     ```
 
+- Submit a job with sentiment analysis
```

Comment:
I think it's fine for now, but we should think at some point if it still makes sense to have all these examples in the README (@TudorCRL @nickgerig )